### DISCUSSION TITLE
is there an event i can log to identify who spawned a big cage on me in freemode?

### DISCUSSION THREAD

**Discussion by lonelybud** [YimMenu/YimMenu#2692] [Q&A] [answered]
in public session a player spawned a big cage on me and it was like difficult to identify ...
I can then somehow know the hash of that object and later filter out to identify.

**Comment by maybegreat48** [comment, marked as answer]
You can try, but you'll always end up with false positives since spawned objects usually bounce around multiple clients before reaching your machine

**Reply by lonelybud** [reply]
lol
